### DISCUSSION TITLE
Vercel 部署MongoDB 数据库地址如何填写

### DISCUSSION THREAD

**Discussion by szsyzx** [Qexo/Qexo#227] [General]
MongoDB 数据库地址如何填写
![image](https://user-images.githubusercontent.com/21995452/192711742-7149c20f-fd2f-40db-8b8a-43744e6f3067.png)
跟教程显示不一样
![image](https://user-images.githubusercontent.com/21995452/192711891-51bcd9c2-d95b-43e4-bb79-c86f361ec723.png)

**Comment by am-abudu** [comment]
mongodb+srv://cluster0.ttd0vof.mongodb.net
